client/src/components/FileCard.css

```diff
@@ -0,0 +1,27 @@
+.file__info {
+  display: flex;
+  background-color: rgb(156, 218, 220);
+  align-items: center;
+  column-gap: 10px;
+  width: 100%;
+  padding: 2px;
+  border: 1px solid rgb(147, 143, 229);
+  border-radius: 10px;
+  margin: 10px;
+}
+
+.file__name {
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.file__subtitle {
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.file__description--container {

```

Comment:
i would suggest not to mess up underscores `_` and hyphens `-` in class name. and there is no need to put double underscores either. better use just hyphens `-` in order to separate words in your css class names.